Cargo.toml

```diff
@@ -19,14 +19,14 @@ bb8 = { version = "0.9", optional = true }
 bb8-postgres = { version = "0.9", optional = true }
 bigdecimal = "0.4"
 byteorder = "1.5.0"
-chrono = "0.4"
+chrono = ">= 0.4.34, < 0.4.40"
 dashmap = "6.1.0"
 datafusion = { version = "44", default-features = false }
 datafusion-federation = { version = "=0.3.4", features = [
   "sql",
 ], optional = true }
 datafusion-proto = { version = "44", optional = true }
-duckdb = { version = "1.1.1", features = [
+duckdb = { version = "<1.2.0", features = [
```

Comment:
duckdb-rs 1.2.0 has Linux build issue (should upgrade once 1.2.0 is available) and also requires Arrow 54 - should be upgrade in the same time